framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/scenario/OpenShiftScenario.java

```diff
@@ -246,9 +250,27 @@ public void afterScenarioFinished(T deploymentScenario) {
     @Override
     public MavenRepositoryDeployment getMavenRepositoryDeployment() {
         return externalDeployments.stream().filter(deployment -> ExternalDeploymentID.MAVEN_REPOSITORY.equals(deployment.getKey()))
-                                           .map(ExternalDeployment::getDeploymentInformation)
```

Comment:
My formatter automatically changes this.... can you check if it's doing the same in your IDE?

Reply:
probably yes, but I prefer the former formatting (even though made manually) for better readability.

Reply:
I changed the confirmation on my IDE so it won't change these lines again.